instrumentation/opentelemetry-instrumentation-asgi/tests/test_asgi_middleware.py

```diff
@@ -1692,21 +1683,12 @@ def test_query_string(self):
             attrs[SpanAttributes.HTTP_URL], "http://127.0.0.1/?foo=bar"
         )
 
-    def test_query_string_new_semconv(self):
```

Comment:
Why remove the test? Can't we adapt it to get the query string using other attributes?

Reply:
Great suggestion. Would self.assertEqual(attrs[SpanAttributes.URL_PATH], "/") and self.assertEqual(attrs[SpanAttrbiutes.SERVER_ADDRESS],"127.0.0.1") work?